experimental/traffic-portal/src/app/shared/generic-table/generic-table.component.scss

```diff
@@ -22,6 +22,7 @@ ag-grid-angular {
 	width: 80vw;
 	display: flex;
 	margin: auto;
+	overflow-x: scroll;
```

Comment:
this will make a horizontal scrollbar permanently appear on all `.extra-actions`s - is that what we really want? Or should this be `auto`?

Note that I couldn't reproduce that in Firefox, which seems to ignore the spec and treat `scroll` the same as `auto`, but in Chrome the scrollbar was properly permanent.

Reply:
I didn't actually mean to commit this change. Was trying to fix an issue with content cutting off on smaller screens but that issue is much more endemic then just this change anyways.